### ISSUE TITLE
some segmented data not found for unmatched tumours

### ISSUE THREAD

**Issue by rdmorin** [morinlab/GAMBLR#19]
Some of the controlfreec results seem to be missing from GAMBLR. It might be that hg38 lifted segmented data from controlfreec were never loaded into the database. Is this a new thing?

```
all_segs_unmatched %>% pull(ID) %>% unique()
  [1] "00-14595_tumorC"           "00-15201_tumorA"           "00-15201_tumorB"          
  [4] "00-23442_tumorA"           "00-23442_tumorB"           "00-26427_tumorA"          
  [7] "01-12047_tumorB"           "01-12047_tumorC"           "01-14774_tumorA"          
 [10] "01-14774_tumorB"           "01-16433_tumorA"           "01-16433_tumorB"          
 [13] "01-20774T"                 "01-23117_tumorA"           "01-23117_tumorB"          
 [16] "01-23942_tumorA"           "01-23942_tumorB"           "01-28152_tumorA"          
 [19] "01-28152_tumorB"           "02-13135T"                 "02-15630_tumorA"          
 [22] "02-15630_tumorB"           "02-18356_tumorA"           "02-18356_tumorB"          
 [25] "02-20170T"                 "02-22991T"                 "02-24492_tumorA"          
 [28] "02-28397_tumorA"           "02-28397_tumorB"           "03-10363T"                
 [31] "03-10440_tumorA"           "03-10440_tumorB"           "03-11110T"                
 [34] "03-19103_tumorB"           "03-23488_tumorA"           "03-23488_tumorB"          
 [37] "03-33266_tumorA"           "03-33266_tumorB"           "04-14066_tumorA"          
 [40] "04-14066_tumorB"           "04-14093_tumorA"           "04-14093_tumorB"          
 [43] "04-21856_tumorA"           "04-24061_tumorA"           "04-24061_tumorB"          
 [46] "04-24937T"                 "04-28140T"                 "04-35039T"                
 [49] "05-12939T"                 "05-17793T"                 "05-18426T"                
 [52] "05-21634T"                 "05-22052T"                 "05-23110T"                
 [55] "05-24065T"                 "05-24395T"                 "05-24401T"                
 [58] "05-24666T"                 "05-25439T"                 "05-25674T"                
 [61] "05-32150T"                 "05-32150_tumorA"           "05-32150_tumorB"          
 [64] "05-32762T"                 "05-32947T"                 "06-10398T"                
 [67] "06-11535T"                 "06-11677_tumorA"           "06-11677_tumorB"          
 [70] "06-14634T"                 "06-15256T"                 "06-15922T"                
 [73] "06-16716T"                 "06-19919T"                 "06-22057T"                
 [76] "06-22314_tumorA"           "06-22314_tumorB"           "06-23907T"                
 [79] "06-24255_tumorC"           "06-24255_tumorD"           "06-24915T"                
 [82] "06-24925T"                 "06-25674T"                 "06-30025T"                
 [85] "06-30145T"                 "06-33777T"                 "06-34043T"                
 [88] "07-10483T"                 "07-17613T"                 "07-25012T"                
 [91] "07-25994_tumorB"           "07-25994_tumorC"           "07-30628T"                
 [94] "07-31833T"                 "07-35482T"                 "07-40648_tumorA"          
 [97] "07-40648_tumorB"           "07-41887_tumorA"           "07-41887_tumorB"          
[100] "08-10249_tumorA"           "08-10249_tumorB"           "08-13706T"                
[103] "08-15460T"                 "08-15460_tumorA"           "08-15460_tumorB"          
[106] "08-15555T"                 "08-17645_tumorB"           "08-19764T"                
[109] "08-25894T"                 "08-29440_tumorA"           "08-29440_tumorB"          
[112] "08-33625T"                 "09-11467T"                 "09-12737T"                
[115] "09-12864T"                 "09-15842_tumorA"           "09-15842_tumorB"          
[118] "09-16981T"                 "09-21480T"                 "09-27193T"                
[121] "09-31008_tumorA"           "09-31008_tumorB"           "09-31233T"                
[124] "09-31895T"                 "09-33003T"                 "09-33003_tumorA"          
[127] "09-33003_tumorB"           "09-41082T"                 "09-41114T"                
[130] "10-10826T"                 "10-11584_tumorA"           "10-11584_tumorB"          
[133] "10-15025T"                 "10-27154T"                 "10-28165T"                
[136] "10-31625T"                 "10-32847T"                 "10-36955_tumorA"          
[139] "10-36955_tumorB"           "10-40676T"                 "10-41170T"                
[142] "11-14210T"                 "11-21727T"                 "11-35935T"                
[145] "12-17272_tumorA"           "12-17272_tumorB"           "12-29259T"                
[148] "13-22818T"                 "13-26601T"                 "13-26835_tumorB"          
[151] "13-30451T"                 "13-31210T"                 "13-38657_tumorA"          
[154] "13-38657_tumorB"           "13-42815T"                 "14-10498_tumorA"          
[157] "14-10498_tumorB"           "14-11247T"                 "14-13959T"                
[160] "14-14094T"                 "14-16281T"                 "14-16707T"                
[163] "14-20552_tumorA"           "14-20552_tumorB"           "14-20962T"                
[166] "14-23891T"                 "14-24648_tumorA"           "14-24648_tumorB"          
[169] "14-25466T"                 "14-27873T"                 "14-29443_tumorB"          
[172] "14-32442T"                 "14-33262T"                 "14-33436T"                
[175] "14-35026T"                 "14-35472_tumorA"           "14-35472_tumorB"          
[178] "14-35632T"                 "14-36022T"                 "14-37722T"                
[181] "14-41461T"                 "15-10535T"                 "15-11617T"                
[184] "15-13365T"                 "15-13383_tumorB"           "15-15757T"                
[187] "15-21654T"                 "15-24058T"                 "15-24306T"                
[190] "15-26538T"                 "15-29858T"                 "15-31924T"                
[193] "15-34472T"                 "15-38154T"                 "15-43657T"                
[196] "15-43891T"                 "16-11636T"                 "16-12281_tumorB"          
[199] "16-13732T"                 "16-16192T"                 "16-16723T"                
[202] "16-17861T"                 "16-18029T"                 "16-18623T"                
[205] "16-23208T"                 "16-27074_tumorA"           "16-27074_tumorB"          
[208] "16-27413T"                 "16-29329T"                 "16-31791T"                
[211] "16-37587_tumorB"           "17-12136T"                 "17-23504T"                
[214] "17-23711_tumorA"           "17-23711_tumorB"           "17-33596_tumorA"          
[217] "17-33596_tumorB"           "17-36275T"                 "17-36275_tumorB"          
[220] "17-40409_tumorA"           "17-40409_tumorB"           "76-10146_tumorA"          
[223] "76-10146_tumorB"           "81-52884T"                 "89-62169T"                
[226] "92-38267_tumorA"           "92-38267_tumorB"           "92-38626_tumorA"          
[229] "92-38626_tumorB"           "94-15772_tumorA"           "94-15772_tumorB"          
[232] "94-25764_tumorB"           "94-26795T"                 "95-32141_tumorA"          
[235] "95-32141_tumorB"           "95-32814T"                 "96-31596T"                
[238] "97-14402T"                 "97-18502_tumorA"           "97-18502_tumorB"          
[241] "98-22532T"                 "99-13280T"                 "99-27137T"                
[244] "DOHH-2"                    "Farage"                    "FL1001T2"                 
[247] "FL1002T2"                  "FL1003T2"                  "FL1004T2"                 
[250] "FL1005T2"                  "FL1006T2"                  "FL1007T2"                 
[253] "FL1008T2"                  "FL1010T2"                  "FL1012T2"                 
[256] "FL1013T2"                  "FL1015T2"                  "FL1016T2"                 
[259] "FL1018T2"                  "FL1019T2"                  "FL1020T2"                 
[262] "HBL-1"                     "HT"                        "HTMCP-01-01-00003-01D-03D"
[265] "HTMCP-01-01-00012-01A-01D" "HTMCP-01-01-00451-01A-01D" "HTMCP-01-02-00013-01A-01D"
[268] "HTMCP-01-02-00017-01A-01D" "HTMCP-01-06-00036-01E"     "HTMCP-01-06-00105-01A-01D"
[271] "HTMCP-01-06-00146-01A-01D" "HTMCP-01-06-00175-01A-01D" "HTMCP-01-06-00185-01A-01D"
[274] "HTMCP-01-06-00206-01A-01D" "HTMCP-01-06-00227-01A-01D" "HTMCP-01-06-00232-01A-01D"
[277] "HTMCP-01-06-00242-01A-01D" "HTMCP-01-06-00253-01A-01D" "HTMCP-01-06-00255-01A-01D"
[280] "HTMCP-01-06-00299-01A-01D" "HTMCP-01-06-00306-01A-01D" "HTMCP-01-06-00307-01A-01D"
[283] "HTMCP-01-06-00310-01B-01D" "HTMCP-01-06-00314-01A-01D" "HTMCP-01-06-00419-01B-01D"
[286] "HTMCP-01-06-00422-01A-01D" "HTMCP-01-06-00443-01A-01D" "HTMCP-01-06-00485-01A-01D"
[289] "HTMCP-01-06-00497-01A-01D" "HTMCP-01-06-00500-01A-01D" "HTMCP-01-06-00526-01A-01D"
[292] "HTMCP-01-06-00563-01A-01D" "HTMCP-01-06-00594-01A-01D" "HTMCP-01-06-00606-01A-01D"
[295] "HTMCP-01-06-00611-01A-01D" "HTMCP-01-06-00634-01A-01D" "HTMCP-01-07-00336-01A-01E"
[298] "HTMCP-01-10-00160-01A-01D" "HTMCP-01-10-00778-01A-01D" "HTMCP-01-15-00366-01A-01E"
[301] "HTMCP-01-15-00367-01A-01E" "HTMCP-01-15-00370-01A-01E" "HTMCP-01-16-00265-01A-01E"
[304] "HTMCP-01-20-00272-01A-01E" "Karpas422"                 "MD903"                    
[307] "OCI-Ly10"                  "OCI-Ly3"                   "SU-DHL-10"                
[310] "SU-DHL-4"                  "SU-DHL-5"                  "SU-DHL-6"                 
[313] "SU-DHL-9"                  "Toledo"                    "WSU-NHL"  
```

**Comment by Kdreval** [comment]
This is now addressed in #86 by implementing the flatfile functionality

**Comment by Kdreval** [comment]
Closing this issue as resolved :white_check_mark:
